scripts/install-zowe.sh

```diff
@@ -493,13 +500,48 @@ else #not SMPE
     echo
   fi
   echo
+
 fi #End SMPE if
 
-DATA_SET_PREFIX=$USER.ZWE
-if [[ "$CI_IS_SMPE" = "yes" ]]; then
-  DATA_SET_PREFIX=$USER.SMPE
+# Copy xmem server PROCLIB, PARMLIB and LOADLIB memers into targets
+echo "calling zowe-install-proc.sh with"
+echo "    ZOWE_DSN_PREFIX=${DATA_SET_PREFIX}"
+echo "    ZOWE_SERVER_PROCLIB_DSNAME=$CIZT_PROCLIB_DS"
+cd $CIZT_ZOWE_ROOT_DIR/scripts/utils
+RUN_SCRIPT="./zowe-install-proc.sh ${DATA_SET_PREFIX} $CIZT_PROCLIB_DS"
+run_script_with_timeout "$RUN_SCRIPT" 3600
+EXIT_CODE=$?
+if [[ "$EXIT_CODE" != "0" ]]; then
+  echo "[${SCRIPT_NAME}][error] ${RUN_SCRIPT} failed."
 fi
 
+echo "calling zowe-install-xmem.sh with"
+echo "  datasetprefix ${DATA_SET_PREFIX}"
+echo "  zss loadlib   ${CIZT_ZSS_LOADLIB_DS_NAME}"
+echo "  zss parmlib   ${CIZT_ZSS_PARMLIB_DS_NAME}"
+echo "  zss proclib   ${CIZT_ZSS_PROCLIB_DS_NAME}"
+
+ls -l     ./zowe-install-xmem.sh # check
+chmod a+x ./zowe-install-xmem.sh 
```

Comment:
Please remove this line - if the script doesn't have execute permission, we need to know about it as a install defect